### PR TITLE
Fix missing created_at in TokenGeneration

### PR DESCRIPTION
**Pull request by k9845**
Addresses
- https://github.com/IFRCGo/go-api/issues/2147
- https://github.com/IFRCGo/go-web-app/issues/1009

## Changes
- Add `created_at` in validated_data

## Checklist
Things that should succeed before merging.

- [ ] Updated/ran unit tests
- [ ] Updated CHANGELOG.md

## Release

If there is a version update, make sure to tag the repository with the latest version.

### CODE REVIEW

File: registrations/serializers.py [hunk 1]
```diff
@@ -194,7 +194,7 @@ def create(self, validated_data):
         # Check if private and public key exists
         if not (settings.JWT_PRIVATE_KEY and settings.JWT_PUBLIC_KEY):
             raise serializers.ValidationError('Please contact system adminstrators to configurate private and public key.')
-
+        validated_data['created_at'] = timezone.now()
         instance = super().create(validated_data)
```
Comment (thenav56) [inline registrations/serializers.py:198]:
```suggestion
        instance = super().create(validated_data)
        validated_data['created_at'] = instance.created_at
```

**Review by thenav56 — APPROVED** [review]
LGTM

1 minor change

**Comment by k9845** [comment]
@szabozoltan69  let's merge and deploy this to staging.

**Review by susilnem — APPROVED** [review]

**Comment by k9845** [comment]
@szabozoltan69 Can you please merge this to staging?

**Review by szabozoltan69 — APPROVED** [review]

**Pull request merged**